pkgdb2/ui/packages.py

```diff
@@ -878,6 +880,19 @@ def package_unretire(namespace, package, full=True):
                     'error')
                 break
 
+        # Check the review URL
+        bz = APP.config.get('PKGDB2_BUGZILLA_URL')
+        try:
+            int(review_url)
+            review_url = bz + '/' + review_url
+        except (TypeError, ValueError):
+            pass
+        if review_url and bz not in review_url:
```

Comment:
Also, we might be able to support both the /show_bug.cgi?id=123456 and /123456 urls, and always display the /123456?